src/Components/TestExplorer.fs

```diff
@@ -1544,7 +1579,12 @@ module Interactions =
                     logger.Error(message, buildFailures |> List.map ProjectPath.fromProject)
 
                 else
-                    let librariesCapableOfListOnlyDiscovery = set [ "Expecto"; "xunit.abstractions" ]
+                    let detectablePackageToFramework =
```

Comment:
xUnit and expecto can be discovered with `--list-tests` so we need to set the test framework in this discovery method too